gp-conditional-pricing/gpcp-product-pricing-labels.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Gravity Perks // GP Conditional Pricing // Display Price Labels (JS)
+ * http://gravitywiz.com/documentation/gravity-forms-conditional-pricing/
+ *
+ * This snippet requires the PHP counterpart gpcp-product-pricing-labels.php
+ */
+( function( $ ) {
+	function update_price_lables() {
+		$( 'label[data-gpcp-template], option[data-gpcp-template]' ).each( function() {
+			var $priceElem = $( this ).is( 'option' ) ? $( this ) : $( this ).siblings( 'input' ),
+				price = gformFormatMoney( $priceElem.val().split( '|' )[1] ),
+				template = $( this ).attr( 'data-gpcp-template' );
+			$( this ).html( template.replace( '{price}', price ) );
+		} );
+	}
+	gform.addAction( 'gpcp_after_update_pricing', update_price_lables);
+	update_price_lables();
```

Comment:
Is the `gpcp_after_update_pricing` filter not triggered on the first render? I would have expected it to...

If not, I like your solution. Small type in the function name: "labels" vs "lables".

Reply:
It is, but there was an issue if the Conditional Pricing rule was set to a Date field that was populated using {date_mdy}. 

https://secure.helpscout.net/conversation/1254561365/19350/